PyExpLabSys/drivers/srs_sr830.py

```diff
@@ -38,22 +43,30 @@ def use_external_reference(self, trigger: str):
         self.scpi_comm('RSLP {}'.format(param))
         return True
 
-    def use_internal_freq_reference(self, frequency: float, amplitude: float = 1):
+    def use_internal_freq_reference(self, frequency: float, amplitude: float = None):
         self.scpi_comm('FMOD 1')  # Set frequency reference to internal
         self.scpi_comm('FREQ {}'.format(frequency))
-        self.scpi_comm('SLVL {}'.format(amplitude))
+        if amplitude is not None:
+            self.scpi_comm('SLVL {}'.format(amplitude))
 
     def sensitivity(self, sensitivity: int = None):
         """
-         0: 2 nV/fA       1: 5 nV/fA         2: 10 nV/fA       3: 20 nV/fA
```

Comment:
Did "0" mean the range was set to 2 nV or 2 fA depending on some other setting?  And you focus only on the voltage now?

Reply:
The nv/fA is an error in the manual that I originally copied into the code. It should just read 2nV full-scale.